test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SearchCommandTests.cs

```diff
@@ -0,0 +1,951 @@
+
+                server.Get.Add("/v3/index.json", r => index);
+
+                string queryResult = $@"
+                {{
+                    ""@context"":
+                    {{
+                        ""@vocab"": ""http://schema.nuget.org/schema#"",
+                        ""@base"": ""https://api.nuget.org/v3/registration5-semver1/""
+                    }},
+                    ""totalHits"": 396,
+                    ""data"": [
+                    {{
+                        ""@id"": ""https://api.nuget.org/v3/registration5-semver1/newtonsoft.json/index.json"",
+                        ""@type"": ""Package"",
+                        ""registration"": ""https://api.nuget.org/v3/registration5-semver1/newtonsoft.json/index.json"",
+                        ""id"": ""Fake.Newtonsoft.Json"",
+                        ""version"": ""12.0.3"",
+                        ""description"": ""Json.NET is a popular high-performance JSON framework for .NET"",
+                        ""summary"": """",
+                        ""title"": ""Json.NET"",
+                        ""iconUrl"": ""https://api.nuget.org/v3-flatcontainer/newtonsoft.json/12.0.3/icon"",
+                        ""licenseUrl"": ""https://www.nuget.org/packages/Newtonsoft.Json/12.0.3/license"",
+                        ""projectUrl"": ""https://www.newtonsoft.com/json"",
+
+                        ""tags"": [
+                            ""json""
+                        ],
+
+                        ""authors"": [
+                        ""James Newton-King""
+                        ],
+
+                        ""totalDownloads"": 531607259,
+                        ""verified"": true,
+
+                        ""packageTypes"": [
+                        {{
+                            ""name"": ""Dependency""
+                        }}
+                        ],
+
+                        ""versions"": [
+                        {{
+                            ""version"": ""3.5.8"",
+                            ""downloads"": 461992,
+                            ""@id"": ""https://api.nuget.org/v3/registration5-semver1/newtonsoft.json/3.5.8.json""
+                        }}
+                        ]
+                    }}
+                    ]
+                }}";
+
+                server.Get.Add("/search/query?q=json&skip=0&take=20&prerelease=false&semVerLevel=2.0.0", r => queryResult);
+
+                server.Start();
+
+                // Act
+                string[] args = new[]
+                {
+                    "search",
+                    "json",
+                };
+
+                CommandRunnerResult result = CommandRunner.Run(
+                    nugetexe,
+                    config.WorkingDirectory,
+                    string.Join(" ", args),
+                    waitForExit: true);
+
+                server.Stop();
+
+                // Assert
+                Assert.True(0 == result.Item1, $"{result.Item2} {result.Item3}");
```

Comment:
Anywhere you have Item1/Item2/Item3, could you change them to their counterparts: Success/Error/AllOutput? I've been trying to move us away from these variables.